svf/include/SVFIR/SVFValue.h

```diff
@@ -315,21 +316,21 @@ class SVFFunction : public SVFValue
     const SVFFunctionType* funcType; /// FunctionType, which is different from the type (PointerType) of this SVFFunction
     SVFLoopAndDomInfo* loopAndDom;  /// the loop and dominate information
     const SVFFunction* realDefFun;  /// the definition of a function across multiple modules
-    std::vector<const SVFBasicBlock*> allBBs;   /// all BasicBlocks of this function
     std::vector<const SVFArgument*> allArgs;    /// all formal arguments of this function
     SVFBasicBlock *exitBlock;             /// a 'single' basic block having no successors and containing return instruction in a function
     const CallGraphNode *callGraphNode;          /// call graph node for this function
+    BasicBlockGraph* bbGraph; /// the basic block graph of this function
 
 protected:
     inline void setCallGraphNode(CallGraphNode *cgn)
     {
         callGraphNode = cgn;
     }
 
-    ///@{ attributes to be set only through Module builders e.g., LLVMModule
-    inline void addBasicBlock(const SVFBasicBlock* bb)
-    {
-        allBBs.push_back(bb);
+
+    inline SVFBasicBlock* addBasicBlock(const std::string& bbName) {
```

Comment:
remove this and use bbGraph::addBasicBlock.

Reply:
ok